data/json/monsters/zed_nomex.json

```
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "mon_zombie_nomex",
+    "type": "MONSTER",
+    "name": { "str": "nomex zombie" },
```

```suggestion
    "name": { "str": "Nomex zombie" },
```
*Angry DuPont noises*